tests/unit/config/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# sACN config unit tests
+
+add_subdirectory(source_disabled)
```

Comment:
Following from my CMake test comment earlier, I think these config directories should be placed next to the other ones in the tree:
```
|- tests/
|-- unit/
|-- configs/
|--- dynamic/
|--- static/
|--- source_disabled_dynamic/
|--- source_disabled_static/
```

Reply:
https://github.com/ETCLabs/sACN/pull/49/commits/ea1b6c5288b8eae72c7140acab09f0ebd4b4630e